resources/views/checkout/partials/sections/payment.blade.php

```diff
@@ -2,20 +2,21 @@
     <form id="payment" class="flex flex-col gap-2" v-on:submit.prevent="save(['payment_method'], 4)">
         <div v-for="(method, index) in checkout.payment_methods">
             <x-rapidez-ct::input.radio
-                class="min-h-[40px]"
                 name="payment_method"
                 v-bind:value="method.code"
                 v-bind:dusk="'method-'+index"
                 v-model="checkout.payment_method"
                 required
             >
-                <div>@{{ method.title }}</div>
-                <img
-                    class="max-h-10"
-                    v-bind:alt="method.code"
-                    v-bind:src="`/vendor/payment-icons/${method.code}.svg`"
-                    onerror="this.onerror=null; this.src='/vendor/payment-icons/creditcard.svg'"
-                />
+                <x-slot:wrapper class="min-h-[40px]">
+                    <div>@{{ method.title }}</div>
+                    <img
+                        class="max-h-10"
+                        v-bind:alt="method.code"
+                        v-bind:src="`/vendor/payment-icons/${method.code}.svg`"
+                        onerror="this.onerror=null; this.src='/vendor/payment-icons/creditcard.svg'"
+                    />
+                </x-slot:wrapper>
```

Comment:
When you do this:
```suggestion
                <x-slot:wrapper class="min-h-[40px]"></x-slot:wrapper>
                <div>@{{ method.title }}</div>
                <img
                    class="max-h-10"
                    v-bind:alt="method.code"
                    v-bind:src="`/vendor/payment-icons/${method.code}.svg`"
                    onerror="this.onerror=null; this.src='/vendor/payment-icons/creditcard.svg'"
                />
```
You can just always use `$slot` in the component. Sadly self closing doesn't work:
```
<x-slot:wrapper class="min-h-[40px]"/>
```
Which makes sense as a slot normally does have any content.

Reply:
You already figured this out, should have read the summary first 🙃 I think it's good to keep it consistent and clean by using 1 slot for the content.

Reply:
True but the "complexity" in the component isn't very nice. Do we need the `@if(!$wrapper->isEmpty() || isset($slot))`?  As one of them should always be there, don't see any cases where the radio doesn't have a "label".

And maybe `{{ $wrapper->isEmpty() ? $slot : $wrapper }}` could be `{{ $wrapper ?? $slot }}` which makes it cleaner?

Reply:
`{{ $wrapper ?? $slot }}` is what I had originally, but it doesn't quite do what you want because $wrapper still exists (defined by `@slots()`), it's just empty.

I was thinking I can just do

```blade
{{ $wrapper }}
{{ $slot }}
```

though, maybe that's cleaner?

Reply:
I'll have to try it tomorrow :)

Reply:
🚀

Reply:
It didn't quite work as easily as `{{ $wrapper ?: $slot }}`, but `{!! $wrapper->toHtml() ?: $slot !!}` does the trick :)